src/main/java/uk/gov/indexer/ctserver/CTServer.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.indexer.ctserver;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import java.io.ByteArrayOutputStream;
+import java.security.MessageDigest;
+import java.util.Base64;
+
+public class CTServer {
```

Comment:
I wonder if, longer term, we'll want to extract this into a client library? Not something for this PR anyway.